trustedApplications/trustedApplicationsPane.ts

```diff
@@ -92,28 +109,41 @@ function createApplicationTable(subject) {
     createText('th', 'Access modes'),
     createText('th', 'Actions')
   ])
-  applicationsTable.appendChild(header)
+  applicationsTable.appendChild(header);
```

Comment:
Minor comment: We need to avoid minor diffs in code style such as this (some places we have semi-colon, others not). We've used StandardJS for this before, lets make sure that the same rules apply for our [TypeScript](https://standardjs.com/rules.html) files (or something close enough, at least)

Reply:
I tried to respect the current coding style. In this case, however, the semi-colon can not be removed, because the following statement is enclosed in brackets for the type cast. Otherwise it would interpret that statement as an argument passed to a function that `appendChild` should have removed.

That said, when I ran StandardJS on the JS files, there were quite a lot of errors...

Reply:
Yes, we need a strategy for how to resolve this when migrating JS files.